### ISSUE TITLE
Cannot find native module 'ExponentCamera'

### ISSUE THREAD

**Issue by julisahun** [expo/expo#21761]
### Summary

I was trying to build an expo app with a camera feature, but as soon as I import the expo-camera package, this error occurs

### Managed or bare workflow?

managed

### What platform(s) does this occur on?

Android

### Package versions

```json
"expo": "^48.0.7",
"expo-camera": "^13.2.1",
"expo-dev-client": "~2.1.5",

### Environment

expo-env-info 1.0.5 environment info:
    System:
      OS: Linux 5.10 Debian GNU/Linux 11 (bullseye) 11 (bullseye)
      Shell: 5.8 - /usr/bin/zsh
    Binaries:
      Node: 19.2.0 - /home/linuxbrew/.linuxbrew/bin/node
      Yarn: 1.22.19 - /home/linuxbrew/.linuxbrew/bin/yarn
      npm: 9.3.0 - /home/linuxbrew/.linuxbrew/bin/npm
    npmPackages:
      babel-preset-expo: ^9.3.0 => 9.3.0 
      expo: ^48.0.7 => 48.0.7 
      react: 18.2.0 => 18.2.0 
      react-native: 0.71.3 => 0.71.3 
    npmGlobalPackages:
      eas-cli: 3.8.1
      expo-cli: 6.3.2
    Expo Workflow: managed


### Reproducible demo

```js

import React from 'react'
import { View, Image } from 'react-native'
import { Camera, CameraType } from 'expo-camera';

const App = ({ navigation }) => {
  
  return (
    <View>
    </View>
  )
}

export default App


### Stacktrace (if a crash is involved)

_No response_

**Comment by LuisEGV** [comment]
Had this issue, was able to solve it by locking the expo-camera version to 13.2.0, deleting node-modules, pods and podfile.lock and reinstalling everything. I think it might happened because something gets cashed in the expo-camera modules and it does not reinstall the latest package version thus it creates this error on IOS. However, I am not 100% on the reason why this error can happen. 

TLDR:

Change
`"expo-camera": "13.2.0"`

Delete
- Pod file
- Pod Lock
- Node-Modules

Finally
Install pods and node-modules again.

**Comment by cr7cr8** [comment]
same issue here, once import it  ,it gives Error: Cannot find native module 'ExponentCamera'  ,any solutions ?

**Comment by cr7cr8** [comment]
it turned out you need to install the **expo-camera** package before uploading the project for  the development build

**Comment by EsdrasAlbino** [comment]
fix the issue using _npx expo-doctor_ and following steps to fix depences.

In my case, _expo-camera_ and _react-native_ not was compatible

**Comment by yossydev** [comment]
I am now running into this problem.
I would love to know what your solutions are!

**Comment by MnariMohamed** [comment]
> Lu

I'm wondering why is this getting a downvote although it works
